### DISCUSSION TITLE
odd behavior when customizing with Sass

### DISCUSSION THREAD

**Discussion by benfroelich** [twbs/bootstrap#36954] [Q&A] [answered]
Hi, I got bootstrap 5.2 using npm. my sass cli version is also from npm, 1.54.3 compiled with dart2js 2.17.6.
https://getbootstrap.com/docs/5.0/customize/sass/#importing

When I use option A, "Include all of Bootstrap", it works fine.

When I use option B, "Include parts of Bootstrap", I get this sass error:
```
Error: Undefined variable.
   ╷
20 │   @each $color, $value in $theme-colors-rgb {
   │                           ^^^^^^^^^^^^^^^^^
   ╵
  node_modules/bootstrap/scss/_root.scss 20:27  @import
  source/stylesheets/brand.scss 14:9            root stylesheet
```

I tried copying in the import stack from bootstrap.scss and it worked. This is what the docs recommend in the paragraph following option B.

Is the option B example correct?

Warm regards,
Ben

**Comment by damiankomonski** [comment]
Option B from documentation is correct. You get Error "Undefined variable" because SASS compiler doesn't see this variable.

**Comment by mdo** [comment, marked as answer]
Make sure you read the release migration guide or blog post—you're likely missing the new `_maps.scss` file.

**Reply by benfroelich** [reply]
Yes, somehow I had been using the 5.0 documentation, as the link in my original post says, while I was using 5.2. Option B
works perfectly fine if I use the right documentation version [embarrased].
Thanks for your assistance and sorry for the silly mistake!

**Reply by benfroelich** [reply]
I needed to go to all versions to get to 5.2 or just change my browser url from 5.0 to 5.2.
![image](https://user-images.githubusercontent.com/17666967/185227459-1437809d-7405-4836-b71d-10f38d2e03f8.png)

**Reply by mdo** [reply]
Bootstrap 5.1.x and earlier do not use a dedicated maps stylesheet. This was introduced in the v5.2.0-beta1 release. Here's the main Bootstrap imports in v5.1.3: https://github.com/twbs/bootstrap/blob/v5.1.3/scss/bootstrap.scss.

**Reply by thecristen** [reply]
Total mistake on my part, turns out we were updated to 5.2 -- thanks @mdo 👍🏼
